### PR TITLE
TAJO-1985: Supporting Metric report via JMX.

### PR DESCRIPTION
**Pull request by keybod**
I added a JMX metric reporter.

### CODE REVIEW

**Comment by jinossy** [comment]
Travis failure is not related this PR
Could you trigger the travis CI?

File: tajo-core/src/main/java/org/apache/tajo/util/metrics/TajoSystemMetrics.java [hunk 1]
```diff
@@ -113,6 +117,9 @@ public void start() {
 
     final String jvmMetricsName = metricsGroupName + "-JVM";
     setMetricsReporter(jvmMetricsName);
+    jmxReporter = JmxReporter.forRegistry(metricRegistry).inDomain("Tajo")
```
Comment (jinossy) [inline tajo-core/src/main/java/org/apache/tajo/util/metrics/TajoSystemMetrics.java]: This line should move into the ’if(!inited) {}’

**Comment by keybod** [comment]
Thank you for reviewing!
I have applied it.

**Comment by jinossy** [comment]
+1 LGTM!
Thanks for your contribution!

**Pull request closed without merging**